header/core/listener.go

```diff
@@ -25,6 +26,9 @@ type Listener struct {
 	bServ     blockservice.BlockService
 	construct header.ConstructFn
 	cancel    context.CancelFunc
+
+	// lastHeaderBroadcasted is the timestamp of the last block served by the broadcaster, used for metrics.
+	lastHeaderBroadcasted int64
```

Comment:
i would call this `lastHeaderBroadcast` bc that implies you're measuring the broadcast time rather than the header height

Reply:
done